dashboards/mongodb/main.json

```diff
@@ -9,7 +9,8 @@
       {
         "type": "timeseries",
         "display": "line",
-        "title": "MongoDB Database Throughput",
+        "title": "Throughput",
+        "description": "Queries performed on each database",
```

Comment:
I want to point out that these changes to the dashboards will not be updated automatically, in case that's the expectation. We have a system to update them, but it has yet to run. It doesn't run on a scheduled interval. This submodule also needs to be updated in the server repo for new dashboards.

Reply:
Yeah :) The changes that I've been making aren't major (the same things are being shown, just with different descriptions or re-arranged) so I think it's fine to document either the before or the after.